app/src/main/java/com/infomaniak/swisstransfer/ui/screen/newtransfer/importfiles/SelectedFilesCard.kt

```diff
@@ -0,0 +1,184 @@
+
+    SwissTransferCard(modifier) {
+        SharpRippleButton(onClick = { /*TODO*/ }) {
+            Text(
+                pluralStringResource(R.plurals.filesCount, fileCount, fileCount),
+                modifier = Modifier.padding(start = Margin.Medium),
+                color = SwissTransferTheme.colors.secondaryTextColor,
+                style = SwissTransferTheme.typography.bodySmallRegular,
+            )
+            Text(
+                "•",
+                modifier = Modifier.padding(horizontal = Margin.Small),
+                color = SwissTransferTheme.colors.secondaryTextColor,
+                style = SwissTransferTheme.typography.bodySmallRegular,
+            )
+            Text(
+                formatSpaceLeft(formattedSizeWithUnits),
+                color = SwissTransferTheme.colors.secondaryTextColor,
+                style = SwissTransferTheme.typography.bodySmallRegular,
+            )
+            Spacer(modifier = Modifier.weight(1f))
+            Icon(
+                imageVector = AppImages.AppIcons.ChevronRightSmall,
+                contentDescription = null,
+                modifier = Modifier.padding(Margin.Medium),
+                tint = SwissTransferTheme.colors.iconColor,
+            )
+        }
+
+        LazyRow(
+            Modifier.fillMaxWidth(),
+            contentPadding = PaddingValues(start = Margin.Medium, end = Margin.Medium, bottom = Margin.Medium),
+            horizontalArrangement = Arrangement.spacedBy(Margin.Medium),
+        ) {
+            item(key = TransferLazyRowKey(TransferLazyRowKey.Type.ADD_BUTTON)) {
+                AddNewFileButton(Modifier.animateItem()) { showUploadSourceChoiceBottomSheet() }
+            }
+
+            items(
+                items = files().asReversed(),
+                key = { TransferLazyRowKey(TransferLazyRowKey.Type.FILE, it.uid) },
+            ) { file ->
+                SmallFileTile(
+                    modifier = Modifier.animateItem(),
+                    file = file,
+                    smallFileTileSize = SmallFileTileSize.LARGE,
+                    onRemove = { removeFileByUid(file.uid) },
+                )
+            }
+        }
+    }
+}
+
+@Composable
+private fun formatSpaceLeft(formattedSizeWithUnits: () -> String): String {
+    val formattedSize = formattedSizeWithUnits()
+    val quantity = LocalContext.current.getQuantityFromFormattedSizeWithUnits(formattedSize)
+    return pluralStringResource(R.plurals.transferSpaceLeft, quantity, formattedSize)
+}
+
+private fun Context.getQuantityFromFormattedSizeWithUnits(formattedSize: String): Int {
+    val sizeParts = formattedSize.split(' ', Typography.nbsp) // Space for languages such as EN and NBSP for languages such as FR
+
+    return if (sizeParts.size == 2) {
+        val local = resources.configuration.getLocales().get(0)
+        val parsedNumber = NumberFormat.getInstance(local).parse(sizeParts[0])
```

Comment:
replace get(0) by `first`

Reply:
LocalList has not .first() extension unfortunately